BioForm/profiles/models.py

```diff
@@ -1,10 +1,14 @@
+from django.contrib.auth.models import User
 from django.db import models
 
 class Bio(models.Model):
+    #user = models.OneToOneField(User, on_delete=models.CASCADE)
     name = models.CharField(max_length=200)
-    username = models.CharField(max_length=200, unique=True)
+    user = models.OneToOneField(User, to_field='id', on_delete=models.CASCADE)
```

Comment:
By default foreign key relationship is made with the primary key(`id`) of another table, no need to mention it explicitly.
```suggestion
    user = models.OneToOneField(User, on_delete=models.CASCADE)
```